android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonWizardFormActivity.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Created by keyman on 04/12/2018.
  */
-public class JsonWizardFormActivity extends JsonFormActivity {
+public class JsonWizardFormActivity extends FormConfigurationJsonFormActivity {
```

Comment:
Do we need this change?

Reply:
Yes, this is to enable form configuration

Reply:
Do we need that for this fix?

Reply:
Not really, I added it coz I noticed we had missed it during the form config upgrade

Reply:
ok, could we please remove it then?